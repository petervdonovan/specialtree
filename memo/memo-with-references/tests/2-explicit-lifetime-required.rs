use memo_with_references::memo;

#[memo(cache)]
fn memome(cache: &str) {}

fn main() {}
