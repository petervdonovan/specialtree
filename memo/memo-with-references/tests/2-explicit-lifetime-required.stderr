error: explicit lifetime required on cache argument span
 --> tests/2-explicit-lifetime-required.rs:4:18
  |
4 | fn memome(cache: &str) {}
  |                  ^^^^
